packages/next/client/image.tsx

```diff
@@ -329,6 +329,20 @@ function handleLoading(
       onLoadingCompleteRef.current({ naturalWidth, naturalHeight })
     }
     if (process.env.NODE_ENV !== 'production') {
+      if (layout === 'raw') {
+        const heightModified =
+          img.height !== parseInt(img.getAttribute('height') || '')
+        const widthModified =
+          img.width !== parseInt(img.getAttribute('width') || '')
+        if (
+          (heightModified && !widthModified) ||
+          (!heightModified && widthModified)
+        ) {
+          warnOnce(
+            `Image with src ${src} has either height or width modified, but not the other. If you use CSS to change the size of your image, use height="auto" or width="auto" to maintain the aspect ratio.`
```

Comment:
This is conflating CSS and HTML, right? Width and height should always be integers and the `height: auto` should be in CSS.

Reply:
Yes, maybe it's not clear enough. Since the first part of the warning specifies CSS, I intended that "use height="auto" would be understood in the context of CSS, too. I'll see if I can clarify that.

Reply:
Ah, I realize the `=` makes it look like an attribute. I've updated it to `Image with src ${src} has either height or width modified, but not the other. If you use CSS to change the size of your image, also include the styles 'height: "auto"' or 'width: "auto"' to maintain the aspect ratio.`